mobify/platform-scaffold
# AMP catch-up with develop

Another catch-up PR. View the most recent commits for the changes I made to make it compatible.

https://github.com/mobify/platform-scaffold/pull/820/commits/41a75b7c131b1d5cdaba56df7a9565570fecdbc0
https://github.com/mobify/platform-scaffold/pull/820/commits/b42f442e591a833d33b879ae737b2511995b9b86

~~**JIRA**: (link to JIRA ticket)~~

## Changes
- Adjusted imports due to a lot of the Integration Manager code moving to the `progressive-web-sdk`
- Adjusted redux store initial state due to store schema changes

## How to test-drive this PR
- `npm run test:all`
- `npm run dev` and check out http://localhost:3000/potions.html and http://localhost:3000/eye-of-newt.html
